cmd/cli/app/profile/list.go

```diff
@@ -76,11 +76,11 @@ func listCommand(ctx context.Context, cmd *cobra.Command, conn *grpc.ClientConn)
 		}
 		cmd.Println(out)
 	case app.Table:
-		table := NewProfileTable()
 		for _, v := range resp.Profiles {
```

Comment:
Could we show something when there are no profiles? The other list commands show an empty table and it would be nice to have consistency.

Reply:
ah! true.

Reply:
@eleftherias  What about an output like this:
```bash
 minder profile list
No config file present, using default values.
+----------------------------------------------------+----------------------------------------------------+
|                  PROFILE SUMMARY                   |                                                    |
+----------------------------------------------------+----------------------------------------------------+
```